indy_common/test/types/test_revoc_def_submit_schema.py

```diff
@@ -9,15 +10,15 @@
     (ISSUANCE_TYPE, ChooseField),
     (MAX_CRED_NUM, IntegerField),
     (PUBLIC_KEYS, AnyMapField),
-    (TAILS_HASH, NonEmptyStringField),
-    (TAILS_LOCATION, NonEmptyStringField),
+    (TAILS_HASH, LimitedLengthStringField),
```

Comment:
Are we sure that tails hash has not more than 256 bits? What if it is SHA-512?

Reply:
Should we drop limitation for this field or add some specific limit? Maybe max(current possible hashes) * 10?